source/module_base/tool_quit.cpp

```diff
@@ -54,9 +54,6 @@ void QUIT(void)
     std::cout<<" See output information in : "<<GlobalV::global_out_dir<<std::endl;
 #endif
 
-#ifdef __MPI
-    MPI_Finalize();
-#endif
```

Comment:
What't the point deleting `MPI_Finalize()`? This violates MPI standard.

Reply:
It will make the program stuck. When some processor running warning_quit and other processors do not ，it will stuck

Reply:
> This violates MPI standard.

I don't think it's good to do so. Any other solution?

Reply:
If users give wrong input parameters, ABACUS should give warning message and quit at once. There is no need to consider other things for example leak of memories. Because ABACUS is killed.